src/ncp/ncp_spinel.cpp

```
@@ -413,6 +413,18 @@ void NcpSpinel::HandleValueIs(spinel_prop_key_t aKey, const uint8_t *aBuffer, ui
         break;
     }
 
+    case SPINEL_PROP_INFRA_IF_SEND_ICMP6:
+    {
+        uint32_t            infraIfIndex;
+        const otIp6Address *destAddress;
+        const uint8_t      *data;
+        uint16_t            dataLen;
+
+        SuccessOrExit(ParseInfraIfIcmp6Nd(aBuffer, aLength, infraIfIndex, destAddress, data, dataLen));
```

The error is discarded.